format.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Formats the files by running pre-commit hooks.
+
+while getopts 'ah' opt; do
+  case "$opt" in
+    a)
+      pip install -q pre-commit
```

Comment:
we need to install pre-commit at each run?

Reply:
I think this is true for CI. Locally, it will be installed once in current env, and then just check pre-commit and related dependency packages.